app/src/main/java/io/intrepid/russell/checkin/CheckInService.java

```diff
@@ -160,33 +167,32 @@ public void onResult(@NonNull Status status) {
     }
 
     private GeofencingRequest getGeofencingRequest() {
+        Geofence.Builder fenceBuilder = new Geofence.Builder()
+                .setTransitionTypes(Geofence.GEOFENCE_TRANSITION_EXIT | Geofence.GEOFENCE_TRANSITION_DWELL)
+                .setExpirationDuration(Geofence.NEVER_EXPIRE)
+                .setLoiteringDelay(LOITERING_DELAY);
+
         return new GeofencingRequest.Builder()
                 .setInitialTrigger(GeofencingRequest.INITIAL_TRIGGER_DWELL | GeofencingRequest.INITIAL_TRIGGER_ENTER)
-                .addGeofence(new Geofence.Builder()
+                .addGeofence(fenceBuilder
                         .setRequestId(LocationData.THIRD_ST.id)
                         .setCircularRegion(LocationData.THIRD_ST.lat, LocationData.THIRD_ST.lng, LocationData.GEOFENCE_RADIUS)
-                        .setTransitionTypes(Geofence.GEOFENCE_TRANSITION_EXIT | Geofence.GEOFENCE_TRANSITION_DWELL)
-                        .setExpirationDuration(Geofence.NEVER_EXPIRE)
-                        .setLoiteringDelay(LOITERING_DELAY)
                         .build())
-                .addGeofence(new Geofence.Builder()
+                .addGeofence(fenceBuilder
                         .setRequestId(LocationData.ROGERS_ST.id)
                         .setCircularRegion(LocationData.ROGERS_ST.lat, LocationData.ROGERS_ST.lng, LocationData.GEOFENCE_RADIUS)
```

Comment:
Does this actually work?  Most builders are one-time-use.  It's also usually a good idea to only use a variable for one purpose, to help prevent confusion.  If the docs say it's OK to reuse a builder, then I guess it's meant to be used this way so don't change it.